lib/chef/knife/vsphere_pool_list.rb

```diff
@@ -15,7 +15,7 @@ class Chef::Knife::VspherePoolList < Chef::Knife::BaseVsphereCommand
   def traverse_folders(folder)
     return if folder.is_a? RbVmomi::VIM::VirtualApp
 
-    puts "#{ui.color("Pool", :cyan)}: "+(folder.path[3..-1].map { |x| x[1] }.* '/') if
+    puts "#{ui.color("Pool", :cyan)}: "+(folder.path[3..-1].map { |x| x[1] unless x[1] == "Resources" }.* '/').squeeze("/") if
```

Comment:
Can you elaborate on why elements are being excluded? I get the code, I'm just not grokking the why as our environment doesn't show any of those elements in the first place.

Code-wise I think this might be an opportunity to clean up this area... Maybe something like
```
if folder.is_a? RbVmomi::VIM::ResourcePool
  pools = folder.path[3..-1].reject { |p| p.first ==Resources" }
  puts "#{ui.color("Pool", :cyan)}: " + pools.map(&:first).join('/')
end
```